node/forest_libp2p/bitswap/build.rs

```diff
@@ -0,0 +1,34 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use protobuf_codegen::Customize;
+use walkdir::WalkDir;
+
+const PROTO_DIR: &str = "proto";
+const CARGO_OUT_DIR: &str = "proto";
+
+fn main() -> anyhow::Result<()> {
+    protobuf_codegen::Codegen::new()
+        .pure()
+        .cargo_out_dir(CARGO_OUT_DIR)
+        .inputs(get_inputs()?.as_slice())
+        .include(PROTO_DIR)
+        .customize(Customize::default().lite_runtime(true))
+        .run()?;
+    Ok(())
+}
+
+fn get_inputs() -> anyhow::Result<Vec<String>> {
```

Comment:
[It seems](https://docs.rs/protobuf-codegen/3.2.0/protobuf_codegen/struct.Codegen.html#method.inputs) the `inputs` will gladly receive a normal `Path`, so converting those into a `String` is redundant. What do you think?

Reply:
Sounds good, fixed